client/main.js

```diff
@@ -15,35 +15,39 @@ function Event({ event }) {
   return <div className={ `event list-group-item ${ eventCategory ? eventCategory.toLowerCase() : null }` }>
     <h2>{ event.title } <EventTime time={event.time} /> </h2>
     <p>
-      <span>{ event.info && event.info.Lokacija ? event.info.Lokacija : null }</span>
-      <span> | </span>
-      <span>{ eventCategory }</span>
+      <h4>{ event.info && event.info.Lokacija ? event.info.Lokacija : null }</h4>
+      <span className="label label-primary">{ eventCategory }</span>
     </p>
     <p dangerouslySetInnerHTML={ eventDescription(event.description) }></p>
     <EventLink link={ event.link } />
   </div>;
 }
 
-function Date({ date }) {
-  const dateEvents = date.events.map(event => <Event event={event} />)
+function Date({ date, events }) {
+  const eventList = events.map(event => <Event event={event} />);
+  const dateChunks = date.split('/');
+  const normalizedDateString = `${dateChunks[1]}/${dateChunks[0]}`
+  const formatedDate = format(normalizedDateString, 'D. MMMM', { locale: hrLocale })
+
   return <div className="date list-group">
     <h1>
-      { format(date.date, 'D. MMMM', { locale: hrLocale })}
+      { formatedDate }
     </h1>
     <div>
-      { dateEvents }
+      { eventList }
     </div>
```

Comment:
Um, why not preserve original html semantics by making this (un)ordered list?
```html
<ol>
  { events.map(event => <li><Event event={event} /></li> }
</ol>
```
Also I don't see any particular benefits of placing out this mapper fn because it is small one anyway.